packages/ferreiro-server/package.json

```
@@ -16,7 +16,7 @@
     "amazon"
   ],
   "engines": {
-    "node": "8.10.0"
+    "node": "8.x"
```

Alternatively, you can use semver ranges to allow 8.0.0 to newer:
```suggestion
    "node": ">=8.0.0"
```

I don't remember correctly if yarn throws an error when you install a dependency that requires Node.js 8.x when you have a newer version (10, for example, which is the LTS version), since 10.x doesn't match 8.x, but it surely does for >=8.0.0!

---

That's an interesting thought! Thanks for pointing out.

Does it make sense to specify >=8.0.0 instead of a specific version like `8.x`? When deploying to Heroku, the server will have a specific version (prob 8.15.0). If any of the dependencies use a version above or below 8 that is a breaking change, should it be the normal behavior to throw an error and prob downgrade/fix the dependency version?

It looks to me that it could be the expected behavior, but happy to know your thoughts about it, maybe there is something i'm missing :)

---

Normally, major versions of Node.js are retrocompatible (11.x is an exception, though, but only for very specific things, specially related to V8's API due to the 7.0 version, and removed some of the long-deprecated APIs).

Given that retrocompatibility nature, most JS libraries and applications made in the times of Node.js 0.12.x, are still working in Node.js 11, and that's a lot of years since then!

And yes, imagine Heroku updates Node.js to v10, as it's the new [LTS](https://github.com/nodejs/Release#release-schedule), `>=8.0.0` matches both `8.x` and `10.x`, and any major version after 8. This change helps you as you don't have to deploy a quick patch to bump the version.

---

Nice!! Thanks for the explanation!! It looks to me that it may be safe to make this diff: https://github.com/ferreiro/website/issues/42

Gonna approve your changes and merge it :)